File: Server/Enums.py
from enum import Enum

Suit = Enum("Suit", [("HEART", 0x00), ("TILE", 0x0D), ("CLOVERS", 0x1A), ("PIKES", 0x27)])

Nominal = Enum("Index", [
    ("TWO", 0x00), ("THREE", 0x01), ("FOUR", 0x02),
    ("FIVE", 0x03), ("SIX", 0x04), ("SEVEN", 0x05),
    ("EIGHT", 0x06), ("NINE", 0x07), ("TEN", 0x08),
    ("JACK", 0x09), ("QUEEN", 0x0A), ("KING", 0x0B),
    ("ACE", 0x0C), ("COUNT", 0x0D)
])

Status = Enum("Status", ["START", "ATTACK", "DEFENSE", "THROWIN", "FINISH"])

File: Assets/C#/Datas/Enums/Enums.cs
using System;
using Unity.VisualScripting;

public enum ETypeGame
{
    usual = 0,
    ThrowIn = 1,
    Transferable = 2
}

public enum ESuit
{
    HEART = 0,
    TILE = 1,
    CLOVERS = 2,
    PIKES = 3
}

public enum EStyle
{
    Base,
    Russian,
    Erotic,
    Forest,
    Sea,
    Scary,
    Heroes,
    Fallout,
    Cars
}

public enum EStatus
{
    Null,
    Fold,
    Grab,
    Pass
}

public enum ESortTrump
{
    General,
    Left,
    Right
}

enum CardOrderMethod
{
    SuitThenKind,
    KindThenSuit,
}

public enum EScreens
{
    StartScreen,
    PolicyScreen,
    SignInScreen_NameAvatar,
    SignInScreen,
    MenuScreen,
    ShopScreen,
    RatingScreen,
    CollectionsScreen,
    RewardsScreen,
    SettingsScreen,
    SkinsScreen,
    LoginScreen,
    Count
}

public enum ERole
{
    main = 0,
    firstThrower = 1,
    thrower = 2
}

public enum ENominal
{
    TWO = 0,
    THREE = 1,
    FOUR = 2,
    FIVE = 3,
    SIX = 4,
    SEVEN = 5,
    EIGHT = 6,
    NINE = 7,
    TEN = 8,
    JACK = 9,
    QUEEN = 10,
    KING = 11,
    ACE = 12,
    COUNT = 13
}
public enum LastMove
{
    folding = 0,
    grabbing = 1
}